Gruntfile.js

```diff
@@ -151,7 +153,8 @@ module.exports = function(grunt) {
     imagemin: {
       images: {
         options: {
-          optimizationLevel: 2
+          optimizationLevel: 2,
+          use: [mozjpeg(),pngquant()] //plugins for jpeg & png image compression
```

Comment:
```suggestion
          use: [mozjpeg({quality: 70}), pngquant()] //plugins for jpeg & png image compression
```

Just an edit here to set a constant quality value.

Reply:
Yeah, that's better. Thanks @limzykenneth . I have committed the changes.